test/integration/create-next-app/templates-app.test.ts

```diff
@@ -54,10 +54,13 @@ let testVersion
 describe('create-next-app --app', () => {
   beforeAll(async () => {
     if (testVersion) return
-    const span = new Span({ name: 'parent' })
-    testVersion = (
-      await createNextInstall({ onlyPackages: true, parentSpan: span })
-    ).get('next')
+    // TODO: investigate moving this post publish or create deployed
+    // tarballs to avoid these failing while a publish is in progress
+    testVersion = 'canary'
```

Comment:
Does this need to be `next@canary`?

Reply:
Nevermind, I think the issue is that the env var is missing here:

https://github.com/vercel/next.js/blob/24a146680f6e4cb53737ddee170958b700977ee0/packages/create-next-app/templates/index.ts#L220

It should look like

https://github.com/vercel/next.js/blob/24a146680f6e4cb53737ddee170958b700977ee0/packages/create-next-app/templates/index.ts#L187

Reply:
Added in [a544f2b](https://github.com/vercel/next.js/pull/57025/commits/a544f2b0252929e07f9e41e39e568dcd9f09eee4)